### PR TITLE
Explicitly use underscore and mongo in onTest

### PR DESCRIPTION
**Pull request by eymiha**
Fixes test failure under Meteor 1.2.0.2

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -34,8 +34,10 @@ Package.onTest(function (api) {
   api.use('nimble:restivus');
   api.use('http');
   api.use('coffeescript');
+  api.use('underscore');
   api.use('accounts-base');
   api.use('accounts-password');
+  api.use('mongo', ['client', 'server'])
```
Comment (kahmali) [inline package.js]: The default is for these to be declared on both the client and server, so you can omit the `['client, 'server']` argument.

**Comment by kahmali** [comment]
While you're rebasing the other change, you should remove the punctuation from your commit message's subject line. Helps me sleep better at night :) 

Once those updates are made and you rebase just let me know and we can pull this in. 

Thanks for the PR!

**Comment by kahmali** [comment]
Oh, and if you want to see cool GitHub stuff, include the message "Fix #141" somewhere in your commit message body, so the issue will be automatically closed when we pull this in. Save us the work of cleaning up.

**Comment by eymiha** [comment]
Cool... And I'm testing a fix for #140 that I submitted earlier.

**Comment by eymiha** [comment]
This should take care of the fix... and I incorporated the commit message changes.

**Comment by kahmali** [comment]
Looks good. I'm pulling this in now.

**Pull request merged**